### ISSUE TITLE
Serial.println("'\n'); causes stack overflow

### ISSUE THREAD

**Issue by tjnomad** [arduino/Arduino#2665]
this statement: Serial.println("'\n'); 

causes a stack overflow.  Obviously the missing " is not caught before the line ends!

Your welcome ...TJ

TJ Schmidt: entrepreneur, scientist, creator of wonderful stuff!

**Comment by ffissore** [comment]
It looks like a preprocessor issue. Please give a spin to the version of the IDE linked at #2636 (near the bottom of the conversation)

**Comment by Chris--A** [comment]
This has been resolved, the code makes it to the compiler and produces an error as expected.

IDE 1.5.7
```error
sketch_may21a.ino: In function 'void setup()':
sketch_may21a.ino:6:3: error: 'Serial' was not declared in this scope
sketch_may21a.ino:7:1: error: expected primary-expression before '}' token
sketch_may21a.ino:7:1: error: expected ';' before '}' token
```

IDE 1.6.5 (source build)

```gcc
sketch_may21a.ino:3:3: error: missing terminating " character
sketch_may21a.ino: In function 'void setup()':
sketch_may21a.ino:4:1: error: expected primary-expression before '}' token
sketch_may21a.ino:4:1: error: expected ';' before '}' token
missing terminating " character
```

**Comment by Chris--A** [comment]
Does this still cause problems on operating systems other than windows?

**Comment by ffissore** [comment]
Fixed by #3779

**Comment by m-ri** [comment]
Ciao,

Ho  sentito che stavi cercando  quella roba da un lungo  periodo di tempo,  così l’ho finalmente trovata per te, maggiori informazioni qui  <http://pino.buenaparkdentalspa.com/vwgn>


Grazie per la tua considerazione,marco ribero
